src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC, useCallback, useMemo } from 'react';
+import classNames from 'classnames';
+import { Status } from '../../enum/Status';
+import { Todo } from '../../types/Todo';
+
+interface FooterProps {
+  todos: Todo[],
+  onStatusSelect: (status: Status) => void;
+  todoStatus: string;
+}
+
+export const Footer: FC<FooterProps> = ({
+  todos,
+  onStatusSelect,
+  todoStatus,
+}) => {
+  const handleClick = useCallback((
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+  ) => {
+    onStatusSelect(event.currentTarget.text as Status);
```

Comment:
it's important:
please try to avoid type casting where it possible and think twice if you need it
see my suggestion fo `onClick={handleClick}`  below